pep-0248.txt

```diff
@@ -1,11 +1,11 @@
 PEP: 248
 Title: Python Database API Specification v1.0
-Author: Marc-André Lemburg <mal@lemburg.com>
+Author: Greg Stein <gstein@lyra.org>, Editor: Marc-André Lemburg <mal@lemburg.com>
```

Comment:
"Editor" isn't a currently allowed header, and we've been tightening the validation here -- the simplest solution would be to remove "Editor:" here I'd suggest.

```suggestion
Author: Greg Stein <gstein@lyra.org>, Marc-André Lemburg <mal@lemburg.com>
```

A

Reply:
The problem is that Greg authored the DB-API v1, but I was editor for it after he left. Perhaps I can write "Marc-André Lemburg (editor) <...". Would that work ?

Reply:
I believe it would pass the automatic validation, though it might cause an issue in PEP 0 -- I can't remember if the indexing is on email or name.

I'd suggest try it and check PEP 0 to ensure you're not duplicated!

A

Reply:
The linter doesn't like the parens.

Reply:
All good now.

Reply:
I checked locally with `python -m pre_commit.languages.pygrep --multiline '(?<=\n)Author:(?:(?!((( +|\n {1,8})[^!#$%&()*+,/:;<=>?@\[\\\]\^_`{|}~]+( <[\w!#$%&''*+\-/=?^_{|}~.]+(@| at )[\w\-.]+\.[A-Za-z0-9]+>)?)(,|(?=\n[^ ])))+
\n(?=[A-Z])))' pep-0249.txt` and it is a fairly restrictive pattern -- even square brackets aren't allowed.

Your latest commit (`as Editor`) duplicates you and sorts oddly: https://pep-previews--2889.org.readthedocs.build/ 

`Marc-André Lemburg, Editor` passes and sorts correctly, though still duplicates you.

A